tests/models/decoder_only/language/test_bamba.py

```diff
@@ -0,0 +1,329 @@
+import pytest
+from transformers import AutoModelForCausalLM, AutoTokenizer
+
+from vllm.config import VllmConfig
+from vllm.sampling_params import SamplingParams
+
+from ...utils import check_outputs_equal
+
+# will be ch
+MODELS = ["ibm-fms/Bamba-9.8b-1.8T-hf"]
```

Comment:
The comment trails off, but will there be a small test model available?

Reply:
@raghukiran1224 any plans for a small test model? I think since we do outputs comparison it is not that good to just have a randomly initialised small model

Reply:
@fabianlim @tlrmchlsmth would it be ok to test with a random model or would you rather have a tiny model (say 200M or so) to test with?

Reply:
A tiny model with nonrandom weights would be much better!